server/game/Clocks/Clock.js

```diff
@@ -8,6 +8,20 @@ class Clock {
         this.paused = false;
         this.stateId = 0;
         this.name = 'Clock';
+        this.delayToStartClock = delayToStartClock;
+        this.manuallyPaused = false;
+    }
+
+    manuallyPause() {
+        this.stop();
+        this.manuallyPaused = true;
+        this.updateStateId();
```

Comment:
Manual Pause now just stops the clocks and issues a state update for the client, which will also stop it from counting down in the UI, rather than trying to do weird things with pausing but keeping the initial timestamps.

Stopping the clock calculates how long it's been running for and deducts the appropriate amount of time, which is what we want to happen if you pause.